content/docs/06-integrations/00-kubernetes.md

```diff
@@ -221,37 +385,175 @@ kubeadmconfig:
 | ``kubeadmconfig.kubeletExtraArgs`` | A list of kubelet arguments to set and copy to the nodes.|
 | ``kubeadmconfig.files`` | A list of additional files to copy to the nodes.|
 | ``kubeadmconfig.preKubeadmCommands`` | A list of additional commands to invoke **before** running kubeadm commands.|
 | ``kubeadmconfig.postKubeadmCommands`` | A list of additional commands to invoke **after** running kubeadm commands.|
+| ``kubeadmconfig.clientConfig`` | Settings to manually configure OIDC-based authentication when you choose a third-party (Custom) IDP. Refer to [Configure Custom OIDC and RBAC](/integrations/kubernetes#configurecustomoidcandrbac). |
 | ``pack.serviceDomain`` | The DNS name for the service domain in the cluster. Default: ``cluster.local``.|
 
-## Usage
+## Usage 
   
 The Kubeadm configuration file is where you can do the following:
 
 <br />
 
 - Change the default ``podCIDR`` and ``serviceClusterIpRange`` values. CIDR IPs specified in the configuration file take precedence over other defined CIDR IPs in your environment.
 
-  As you build your cluster, check that the ``podCIDR`` value does not overlap with any hosts or with the service network and the ``serviceClusterIpRange`` value does not overlap with any IP ranges assigned to nodes or pods. For more information, refer to the [Clusters](/clusters) guide and [Cluster Deployment Errors](https://docs.spectrocloud.com/troubleshooting/cluster-deployment).  
+  As you build your cluster, check that the ``podCIDR`` value does not overlap with any hosts or with the service network and the ``serviceClusterIpRange`` value does not overlap with any IP ranges assigned to nodes or pods. For more information, refer to the [Clusters](/clusters) guide and [Cluster Deployment Errors](https://docs.spectrocloud.com/troubleshooting/cluster-deployment). 
 
 
-- Configure OpenID Connect (OIDC) parameters to specify a third-party Identify Provider (IDP). For more information, refer to the [Use RBAC With OIDC](/clusters/cluster-management/cluster-rbac/#userbacwithoidc) guide.
+- Manually configure a third-party OIDC IDP. For more information, check out [Configure Custom OIDC](/integrations/kubernetes#configurecustomoidc). 
 
 
 - Add a certificate for the Spectro Proxy pack if you want to use a reverse proxy with a Kubernetes cluster. For more information, refer to the [Spectro Proxy](/integrations/frp) guide.
 
+
 #### Configuration Changes
 
-The Kubeadm config is updated with hardening improvements, as a pod security policy has been removed.
+The PXK Kubeadm configuration is updated to dynamically enable OIDC based on your IDP selection by adding the ``identityProvider`` parameter. 
+
+<br />
+
+```yaml
+palette:
+   config:
+     dashboard:
+       identityProvider: <your_idp_selection>
+```
+
+<br />
+
+
+### Configure OIDC Identity Provider
+
+Platforms that use PXK can use the OIDC IDP feature. When you add the PXK pack to a cluster profile, Palette displays the following OIDC IDP options. 
```

Comment:
Let's expand a bit on this sentence "Platforms that use PXK can use the OIDC IDP feature". 

This is one of the value propositions of PXK, our flavor of Kubernetes, where we offer a convenient experience for managing OIDC at the Kubernetes layer. Users can leverage Palette as an IDP, and having Palette as an option for the IDP is a benefit because you don't have to worry about configuring a 3rd party IDP. Now, this option is not useful for say mature companies that have their own IDP configured and widely used within their systems, but for smaller organizations or newer companies, this is an attractive option in the early days when capital is limited. 

We also support the ability to easily take advantage of other OIDC providers if they are configured at the Tenant level, or they can add the OIDC configurations to this cluster. 

You can also use this as an opportunity to select a different OIDC if you don't want to use one configured at the tenant level. Say, Okta is the default OIDC idp configured at the tenant level, but for this cluster you want to use Azure AD for X reason.

Reply:
@karl-cardenas-coding Thanks for providing insights into the use case. Revised to say:

"Platforms that use PXK can use the OIDC IDP feature, which offers the convenience of managing OIDC at the Kubernetes layer. The OIDC IDP feature is particularly useful for environments that do not have their own IDP configured. In this scenario, you can leverage Palette as an IDP without having to configure a third-party IDP. We also support the ability to take advantage of other OIDC providers by making it possible for you to configure OIDC at the tenant level. For additional flexibility, if you wish to use a different IDP than the one configured at the tenant level, you can select a different IDP by adding the OIDC configuration to your cluster profile."